opendifferentialprivacy/opendp-ux
# 655 demo data loading

- Added "Setup Demo" button to admin screen: `/admin/dataset/datasetinfo/`
  - Only available to super users when cypress settings are in place
- Demo includes
  - dp_analyst, dp_depositor users
  - dp_analyst - file loading and initial steps already in place
  - credentials added via k8s secrets 
  
---
To test:
- Log in with admin credentials
- Navigate to `/admin/dataset/datasetinfo/`
- Click the "Setup Demo" button
- Log out (or use another browser)
- Log in with "dp_analyst" credentials 
   - See "2" in this [google doc](https://docs.google.com/document/d/1rI78y_9Q73ELl3XwyjNxTeRqydXI0ntZhlo37-gtZw0/edit)
   - There should be a "Teacher Survey" data file "In Progress"
   - Clicking "Continue" goes to the Create Statistics page
